server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexCannedResponse.scala

```diff
@@ -0,0 +1,81 @@
+package uk.gov.ons.addressIndex.server.modules
+
+import uk.gov.ons.addressIndex.crfscala.CrfScala.CrfTokenResult
+import uk.gov.ons.addressIndex.model.server.response._
+
+trait AddressIndexCannedResponse {
+
+  def Limit: Int = 10
+
+  def searchContainerTemplate(
+    tokens: Seq[CrfTokenResult],
+    addresses: Seq[AddressResponseAddress],
+    total: Int
+  ): AddressBySearchResponseContainer = {
+    AddressBySearchResponseContainer(
+      response = AddressBySearchResponse(
+        tokens = tokens.groupBy(_.label).map(x => x._1 -> x._2.map(_.value).mkString(" ")).map(p => CrfTokenResult(p._2, p._1)).toSeq,
+        addresses = addresses,
+        limit = Limit,
+        offset = 0,
+        total = addresses.size
+      ),
+      status = OkAddressResponseStatus
+    )
+  }
+
+  def searchUprnContainerTemplate(optAddresses: Option[AddressResponseAddress]): AddressByUprnResponseContainer = {
```

Comment:
It is reused 2 times and it would be fine without it. [Exaggerated example] Creating an `add(x,y)` function to abstract `1+1` as `add(1, 1)` may be reused many times, but why do it?